experiments/Atmos_GCM/heldsuarez.jl

```diff
@@ -1,61 +1,84 @@
+using Distributions: Uniform
 using LinearAlgebra
 using StaticArrays
+using Random: rand
 using Test
 
 using CLIMA
 using CLIMA.Atmos
 using CLIMA.GenericCallbacks
-using CLIMA.ODESolvers
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.AdditiveRungeKuttaMethod
+using CLIMA.ColumnwiseLUSolver: ManyColumnLU
 using CLIMA.Mesh.Filters
 using CLIMA.Mesh.Grids
 using CLIMA.MoistThermodynamics
 using CLIMA.PlanetParameters
 using CLIMA.VariableTemplates
 
-const p_ground = MSLP
-const T_initial = 255
-const domain_height = 30e3
 
-function init_heldsuarez!(bl, state, aux, coords, t)
-    global p_ground, T_initial
-
-    FT = eltype(state)
+const pressure_ground = MSLP
+const T_init = 300.0 # unit: Kelvin 
+const domain_height = 30e3 # unit: meters
 
-    # TODO: change to altitude function
-    r = norm(coords, 2)
-    h = r - FT(planet_radius)
-    # h = altitude(bl.orientation, aux)
 
-    scale_height = R_d * FT(T_initial) / grav
-    p            = FT(p_ground) * exp(-h / scale_height)
-
-    state.ρ  = air_density(FT(T_initial), p)
+function init_heldsuarez!(bl, state, aux, coords, t)
+    FT            = eltype(state)
+    pressure_sfc  = FT(MSLP)
+    temp_init     = FT(300.0)
```

Comment:
`const T_init` vs `temp_init` within this function - is there a way to keep just one variable with this information ? (We can now pass additional arguments to the solver setup that are made available to the init_heldsuarez! function for example https://github.com/climate-machine/CLIMA/blob/7956017fb59bd1e566716c5d7f8d2816df57d77b/src/Driver/Driver.jl#L212)

Reply:
Agreed. It's odd to have the same temperature be defined twice.